streaming/index.js

```diff
@@ -24,30 +24,15 @@ dotenv.config({
 log.level = process.env.LOG_LEVEL || 'verbose';
 
 /**
- * @param {Object.<string, any>} defaultConfig
- * @param {string} redisUrl
+ * @param {Object.<string, any>} config
  */
-const redisUrlToClient = async (defaultConfig, redisUrl) => {
-  const config = defaultConfig;
-
-  let client;
-
-  if (!redisUrl) {
-    client = redis.createClient(config);
-  } else if (redisUrl.startsWith('unix://')) {
-    client = redis.createClient(Object.assign(config, {
-      socket: {
-        path: redisUrl.slice(7),
-      },
-    }));
-  } else {
-    client = redis.createClient(Object.assign(config, {
-      url: redisUrl,
-    }));
+const createRedisClient = async (config) => {
+  const { redisParams, redisUrl } = config;
+  let client = new Redis(redisParams);
+  if (redisUrl) {
+	client = new Redis(redisUrl, redisParams);
```

Comment:
Indentation issue as well. Not sure why this isn't caught by the CI tests :thinking: